standard.go

```diff
@@ -307,6 +314,41 @@ func (r *StandardRuleset) reduceSnakeHealth(b *BoardState) error {
 	return nil
 }
 
+func (r *StandardRuleset) maybeDamageHazards(b *BoardState) error {
+	for i := 0; i < len(b.Snakes); i++ {
+		snake := &b.Snakes[i]
+		if snake.EliminatedCause != NotEliminated {
+			continue
+		}
+		head := snake.Body[0]
+		for _, p := range b.Hazards {
+			if head == p {
```

Comment:
The check for food below is the only difference in this from the version in `RoyaleRuleset`.